cropper/build.gradle

```diff
@@ -1,6 +1,9 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 //apply from: file('publish.gradle')
```

Comment:
If we will not keep GithubPackages, please delete this

Reply:
👍🏻 , will remove next commit